mpf/devices/multiball.py

```diff
@@ -216,48 +228,49 @@ def add_a_ball(self, **kwargs):
             self.source_playfield.add_ball(balls=1)
 
     @event_handler(9)
-    def start_or_add_a_ball(self, **kwargs):
-        """Start multiball or add a ball if multiball has started."""
+    def event_start_or_add_a_ball(self, **kwargs):
+        """Event handler for start_or_add_a_ball event."""
         del kwargs
+        self.start_or_add_a_ball()
+
+    def start_or_add_a_ball(self):
+        """Start multiball or add a ball if multiball has started."""
         if self.balls_live_target > 0:
             self.add_a_ball()
         else:
             self.start()
 
-    @event_handler(20)
-    def enable(self, **kwargs):
+    def enable(self):
         """Enable the multiball.
 
         If the multiball is not enabled, it cannot start.
-
-        Args:
-            **kwargs: unused
         """
-        del kwargs
+        super().enable()
         self.debug_log("Enabling...")
         self.enabled = True
```

Comment:
This is a `ModeDevice` so it needs its own `self.enabled = True` set here. Would it be advantageous to use `EnableDisableMixin` instead?

Reply:
Yes I guess so.

Reply:
done